### DISCUSSION TITLE
Override Intellicenter limitations while still using it

### DISCUSSION THREAD

**Discussion by shagmwp** [tagyoureit/nodejs-poolController#747] [Q&A]
I have a large pool with 2 pumps/filters. First pump has an Ultratemp heat pump and is Pool only. 2nd pump is pool/spa and has a Mastertemp heater. 

Intellicenter will only let me run one heater at a time as most people who have 2 heaters have them plumbed inline rather than separate pumps.

I would like to use NPC to turn on both heaters and use the temp sensor for "solar" for pump1 and the water temp for pump2.

Is this possible while still keeping everything connected to Intellicenter?

Originally I was hoping to just associate the heat pump with pump 1 and be able to turn it on and off as I pleased, but the bodies of water definitions kind of limit this. 

My second though is to modify Nixie to be both a 1 body of water(pool only) and a 2 body pool/spa, but would I need to bypass Intellicenter at that point?

**Comment by rstrouse** [comment]
Which IntelliCenter model are you running is it an I10D or PS?  You may be able to get there with the rules manager.  Here is another user getting around the single active heater per body
https://github.com/tagyoureit/nodejs-poolController/discussions/741
